Algorithms.Tests/Sequences/A000045_FibonacciSequenceTests.cs

```diff
@@ -5,12 +5,12 @@
 
 namespace Algorithms.Tests.Sequences
 {
-    public class FibonacciSequenceTests
+    public class A000045_FibonacciSequenceTests
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'A000045_FibonacciSequenceTests' to match pascal case naming rules, consider using 'A000045FibonacciSequenceTests'.](https://app.codacy.com/app/siriak/C-Sharp/pullRequest?prid=4650688)